.github/workflows/run-tests.yml

```diff
@@ -20,10 +20,24 @@ on:
 
 jobs:
   run-tests:
-    name: Run tests job
+    name: Run tests job matrix
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        client-type: [go, http]
+        database-type: [badger-file, badger-memory]
+        mutation-type: [gql, collection-named, collection-save]
+        tests: [names, lens, cli]
+
```

Comment:
thought: Once we remove the `-p 1` limitation, we can include the change detector in this file using an `include`, I think the below *might* work:
```
 include:
      - client-type: go
       database-type: badger-memory
        mutation-type: collection-save
        detect-changes: true
 ```

Reply:
Added both the change detector and non change detector to the include so both will now run.